test/feature-benchmark/scenarios.py

```diff
@@ -1482,3 +1482,43 @@ def benchmark(self) -> BenchmarkingSequence:
 """
             ),
         ]
+
+
+class HydrateIndex(ScenarioBig):
+    """Measure the time it takes for an index to hydrate when a cluster comes online."""
+
+    def init(self) -> list[Action]:
+        return [
+            self.table_ten(),
+            TdAction(
+                """
+> CREATE CLUSTER idx_cluster SIZE '16', REPLICATION FACTOR 1
+> CREATE TABLE t1 (f1 INTEGER, f2 INTEGER)
+"""
+            ),
+        ]
+
+    def benchmark(self) -> MeasurementSource:
+        return Td(
+            f"""
+> SET CLUSTER = default
+> DROP INDEX IF EXISTS i1
+> DELETE FROM t1
```

Comment:
I belive you should DROP and recreate the table here instead of DELET-ing it, or move the entire table population in the `init()` section. The reason is that DELETE actually causes all rows to be retracted, resulting in temporary, until compaction runs, double the number of rows in the table. This compounds over each of the successive runs . It is best for every `benchmark()` to start from a clean slate, either an empty table or a pre-populated table.